### ISSUE TITLE
wordgrinder -h error instead of usage text

### ISSUE THREAD

**Issue by a-vrma** [davidgiven/wordgrinder#146]
I'm getting this when running `wordgrinder -h` from git master. This is built against the system LuaJIT:

```
WordGrinder version 0.8 © 2007-2020 David Given
(This version has been compiled with debugging enabled.)
Lua error: [string "src/lua/_prologue.lua"]:52: read from undefined local 'configfile'
stack traceback:
        [C]: in function 'error'
        [string "src/lua/_prologue.lua"]:52: in function '__index'
        [string "src/lua/main.lua"]:281: in function 'fn'
        [string "src/lua/utils.lua"]:400: in function 'ParseArguments'
        [string "src/lua/main.lua"]:364: in function <[string "src/lua/main.lua"]:239>
```

**Comment by a-vrma** [comment]
Also I'm getting `(This version has been compiled with debugging enabled.)` even when running what the file name tells me is the release version. Not sure why.

**Comment by davidgiven** [comment]
Yikes, the -h help text issue was a symptom of a rather more serious problem affecting config files --- thanks for spotting this. Fixing. I'm looking in to the other thing.

**Comment by davidgiven** [comment]
Config file issue fixed in #149

**Comment by davidgiven** [comment]
Yup, it looks like I've been building with debugging enabled for a while now. That actually halves the performance of release builds! So, er, fixed. Thanks!
